src/Products/PloneMeeting/migrations/migrate_to_4217_1.py

```
@@ -33,21 +33,41 @@ def _configureCssTransforms(self):
         logger.info('Done.')
 
+    def _updateGroupsInChargeNotes(self):
+        """Update config and init new fields related to groupsInChargeNotes."""
+        logger.info('Updating "itemFieldsConfig" about groupsInChargeNotes for every MeetingConfigs...')
+        # update MeetingConfig.itemFieldsConfig default for "groupsInChargeNotes"
+        for cfg in self.tool.objectValues('MeetingConfig'):
+            item_fields_config = cfg.getItemFieldsConfig()
+            if not "groupsInChargeNotes" in [row['name'] for row in item_fields_config]:
+                # use default value
+                default_value = cfg.Schema()['itemFieldsConfig'].getDefault(cfg)
+                gic_notes_config = [row for row in default_value if row['name'] == "groupsInChargeNotes"][0]
```

Ruff (0.15.11)

[warning] Prefer `next(...)` over single element slice

Replace with `next(...)`

(RUF015)